web/src/main/java/com/bazaarvoice/emodb/web/auth/AuthorizationConfiguration.java

```diff
@@ -10,6 +10,8 @@
     private final static String DEFAULT_IDENTITY_TABLE = "__auth:keys";
     private final static String DEFAULT_INTERNAL_ID_INDEX_TABLE = "__auth:internal_ids";
     private final static String DEFAULT_PERMISSION_TABLE = "__auth:permissions";
+    private final static String DEFAULT_ROLE_TABLE = "__auth:roles";
+    private final static String DEFAULT_ROLE_GROUP_TABLE = "__auth:role_groups";
 
```

Comment:
My thinking is not that we would ever change them.  Since Emo is open source different installs may have different naming conventions.  I agree, changing the table names on a live system is not supported out-of-the-box and would require outside effort on the administrator's part.